app/components/app/app.e2e.js

```diff
@@ -0,0 +1,19 @@
+describe('App', function() {
```

Comment:
Can we rename it to `app.e2e.ts` in order to have autocompletion? I think we already have protractor type definitions.